vulnerabilities/data_dump.py

```diff
@@ -190,3 +190,44 @@ def npm_dump(extract_data):
                     vulnerability=vulnerability,
                     package=package_fixed
                 )
+
+
+def ruby_dump(extract_data):
+    for package_data in extract_data:
+
+        if package_data['severity']:
```

Comment:
This can be written much shorter and without repetition by using `dict.get()`, since `cvss` is a nullable field in the model:
```
    vulnerability, _ = Vulnerability.objects.get_or_create(
        cve_id=package_data['cve_id'],
        summary=package_data['summary'],
        cvss=package_data.get('severity')
    )
```
But see my other comment below about skipping severity for now.